packages/python/patch-configure

```diff
@@ -0,0 +1,714 @@
+@@ -3269,27 +3272,35 @@
+ fi
+ 
+ # checks for library functions
+-AC_CHECK_FUNCS(alarm accept4 setitimer getitimer bind_textdomain_codeset chown \
+- clock confstr ctermid dup3 execv faccessat fchmod fchmodat fchown fchownat \
+- fexecve fdopendir fork fpathconf fstatat ftime ftruncate futimesat \
++AC_CHECK_FUNCS(alarm accept4 access setitimer getitimer bind bindtextdomain bind_textdomain_codeset chdir chmod chown \
++ clock clock_settime confstr connect ctermid dcgettext dup3 execv execve faccessat fchmod fchmodat fchown fchownat \
++ fexecve fdopendir fopen fork fpathconf freehostent fstatat ftime ftruncate futimesat \
+  futimens futimes gai_strerror getentropy \
+- getgrouplist getgroups getlogin getloadavg getpeername getpgid getpid \
+- getpriority getresuid getresgid getpwent getspnam getspent getsid getwd \
++ getcwd \
++ getgid \
++ getgrouplist getgroups gethostbyaddr getlogin getloadavg getpeername getpgid getpid \
++ getppid \
++ getpriority getresuid getresgid getpwent getspnam getspent getsid getuid getwd \
++ gmtime \
+  if_nameindex \
+- initgroups kill killpg lchmod lchown lockf linkat lstat lutimes mmap \
+- memrchr mbrtowc mkdirat mkfifo \
+- mkfifoat mknod mknodat mktime mremap nice openat pathconf pause pipe2 plock poll \
++ inet_addr \
++ initgroups kill killpg lchmod lchown localtime lockf linkat lstat lutimes mmap \
++ memrchr mbrtowc mkdir mkdirat mkfifo \
++ mkfifoat mknod mknodat mktime mremap nice open openat opendir pathconf pause pipe2 plock poll \
+  posix_fallocate posix_fadvise pread \
+- pthread_init pthread_kill putenv pwrite readlink readlinkat readv realpath renameat \
+- select sem_open sem_timedwait sem_getvalue sem_unlink sendfile setegid seteuid \
++ printf \
++ pthread_init pthread_kill putenv pwrite readlink readlinkat readv realpath rename renameat \
++ rmdir \
++ select sem_open sem_timedwait sem_getvalue sem_unlink sendfile sendto setegid seteuid \
+  setgid sethostname \
+  setlocale setregid setreuid setresuid setresgid setsid setpgid setpgrp setpriority setuid setvbuf \
+  sched_get_priority_max sched_setaffinity sched_setscheduler sched_setparam \
+  sched_rr_get_interval \
+- sigaction sigaltstack siginterrupt sigpending sigrelse \
+- sigtimedwait sigwait sigwaitinfo snprintf strftime strlcpy symlinkat sync \
+- sysconf tcgetpgrp tcsetpgrp tempnam timegm times tmpfile tmpnam tmpnam_r \
+- truncate uname unlinkat unsetenv utimensat utimes waitid waitpid wait3 wait4 \
++ setsockopt \
++ sigaction sigaltstack siginterrupt signal sigpending sigrelse \
++ sigtimedwait sigwait sigwaitinfo snprintf stat strftime strlcpy symlinkat sync \
++ sysconf system tcgetpgrp tcsetpgrp tempnam textdomain timegm times tmpfile tmpnam tmpnam_r \
++ truncate ttyname umask uname unlink unlinkat unsetenv utimensat utimes wait waitid waitpid wait3 wait4 \
+  wcscoll wcsftime wcsxfrm wmemcmp writev _getpty)
+ 
+ AC_CHECK_DECL(dirfd,
+--- Modules/posixmodule.c
```

Comment:
Do you mean the below removals of `#define HAVE_*` and `#if defined(HAVE_*)`? If so then they're here because they're in such close proximity and the patch hunks were clashing